lib/crypto/configure.in

```diff
@@ -106,7 +106,11 @@ elif test "x$with_ssl_zlib" = "xyes" || test "x$with_ssl_zlib" = "x"; then
 			fi
 		done
 		if test "x$SSL_LINK_WITH_ZLIB" = "xno"; then
-		       	AC_MSG_RESULT([no])
+		   	if test "x$with_ssl_zlib" = "xyes"; then
+			   AC_MSG_ERROR([ZLib to use with SSL requested, but not found])
+			else
+			   AC_MSG_RESULT([no])
+			fi
```

Comment:
I do not think we need to support this flag as we will never use OpenSSLs protocol implementation only its cryptolib.